vg/vgpdf/vgpdf_test.go

```diff
@@ -169,3 +171,73 @@ func TestArc(t *testing.T) {
 		t.Fatalf("plot mismatch")
 	}
 }
+
+// Example_multipage shows how one can create a PDF with multiple pages.
+func Example_multipage() {
+	c := vgpdf.New(5*vg.Centimeter, 5*vg.Centimeter)
+
+	{
```

Comment:
Can you do this with a `i, c := range []color.RGBA{{B: 255, A: 255}, {R: 255, A: 255}}`? It would be cleaner and clearer.

Reply:
done.